frontend/packages/dev-console/src/components/import/deploy-image/ResultsDisplay.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { Timestamp, units } from '@console/internal/components/utils';
+import { useFormikContext, FormikValues } from 'formik';
+
+const ResultsDisplay: React.FC = () => {
+  const { values } = useFormikContext<FormikValues>();
+
+  const runsAsRoot = (image) => {
+    const user = _.get(image, 'dockerImageMetadata.Config.User');
+    return !user || user === '0' || user === 'root';
+  };
+
+  const ImagePorts = ({ ports, name }) => (
+    <React.Fragment>
+      {_.size(ports) > 1 ? 'Ports ' : 'Port '}
+      {_.map(ports, (port) => `${port.containerPort}/${port.protocol.toUpperCase()}`).join(
+        ', ',
+      )}{' '}
+      will be load balanced by Service <strong>{values.name || '<name>'}</strong>.
+      <div>
+        Other containers can access this service through the hostname{' '}
+        <strong>{values.name || '<name>'}</strong>.
+      </div>
+    </React.Fragment>
+  );
+
+  return (
+    <div>
+      {!_.isEmpty(values.isi.image) && (
+        <React.Fragment>
+          <div className="co-image-name-results__details">
+            <div className="row">
+              <div className="col-sm-3 col-md-2 hidden-xs text-right h2">
+                <span
+                  className="fa fa-cube text-muted"
+                  style={{ fontSize: '100px' }}
+                  aria-hidden="true"
+                />
+              </div>
+              <div className="col-sm-9 col-md-10">
+                {runsAsRoot(values.isi.image) && (
+                  <div className="alert alert-warning">
```

Comment:
This has dropped the alert change in #1763

Reply:
Good catch. I missed that update, since I started before it.

Reply:
Just noticed that `Alert` is from PF 4. Is that going to be an issue since we still have PF 3 stuff?

Reply:
fixed

Reply:
> Just noticed that `Alert` is from PF 4. Is that going to be an issue since we still have PF 3 stuff?

Yeah, it was part of the effort to move to PF4. We're in a mixed mode already with some PF3 and some PF4. The biggest problem we still have is different font sizes. We're working on addressing that. (It's a bit more involved than overriding the md font size unfortunately.)